contracts/SpokePool.sol

```diff
@@ -839,9 +885,8 @@ abstract contract SpokePool is
         // Exclusivity deadline is inclusive and is the latest timestamp that the exclusive relayer has sole right
         // to fill the relay.
         if (
-            relayData.exclusiveRelayer != msg.sender &&
-            relayData.exclusivityDeadline >= getCurrentTime() &&
```

Comment:
I think we should probably avoid applying these changes in this PR because we're still unsure when it would be merged to production, but other changes on the SVM side will depend on it. OK to have them applied separately in https://github.com/across-protocol/contracts/pull/649 and then base this change on top?

Reply:
It's a bit hard to back out at this point and there will be merge conflicts either way i don't see a reason to back it out now

Reply:
OK; how about getting #649 into master? Then we can rebase this one and resolve the changes. I do think it's important that get #649 in because of other changes that are also touching the same lines.